node/src/chain/acceptor.rs

```diff
@@ -228,6 +229,33 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
                 Ok(txs)
             })?;
 
+            for iter in 0..blk.header().iteration {
+                let mut prov = provisioners_list.clone();
+                let committee_keys =
+                    prov.create_committee(&sortition::Config {
+                        committee_size: SELECTION_COMMITTEE_SIZE,
+                        round: blk.header().height,
+                        seed: blk.header().seed,
```

Comment:
This is wrong. You need `mrb.header().seed`  here.

Reply:
It also returns:
```
{"level":"ERROR","message":"Unable to generate voting committee for missed block","target":"node::chain::acceptor"}
```

Reply:
To resolve this error you need to update eligiblity_flag:

```diff
 
             for iter in 0..blk.header().iteration {
                 let mut prov = provisioners_list.clone();
+                prov.update_eligibility_flag(blk.header().height);
                 let committee_keys =
                     prov.create_committee(&sortition::Config {
                         committee_size: SELECTION_COMMITTEE_SIZE,
                         round: blk.header().height,
-                        seed: blk.header().seed,
+                        seed: mrb.header().seed,
                         step: iter * 3,
                     });
 ``` 

This should fix all issues and generate correct result.

Reply:
Why there is the need of update eligibility flag if the block height is already specified in the sortition config?
Maybe we should change that API (unless there is a use case for calling one of those two methods without the other one)

Reply:
These are distinct scenarios. In essence, for each step within a single round, it is necessary to invoke the `create_committee`, whereas the `update_eligibility_flag` is called only once per round execution.